bin/apisix

```diff
@@ -609,6 +637,22 @@ local function init()
         sys_conf["worker_processes"] = "auto"
     end
 
+    if sys_conf["enable_local_dns"] then
+        local dns_addrs, err = local_dns_resolver("/etc/resolv.conf")
+        if not dns_addrs then
+            error("failed to import local DNS: " .. err)
+        end
+
+        local dns_resolver = sys_conf["dns_resolver"]
+        if dns_resolver then
+            for _, addr  in ipairs(dns_addrs) do
+                table.insert(dns_resolver, addr)
+            end
+        else
+            sys_conf["dns_resolver"] = dns_addrs
```

Comment:
the same as https://github.com/apache/incubator-apisix/pull/1167#discussion_r388035389
why we need this line?

Reply:
There may do not configure 'dns_resolver' in `config.yaml`

Reply:
no test cases cover